### PR TITLE
Adding a few french translation

### PR DESCRIPTION
**Pull request by omahs**
very commit

### CODE REVIEW

**Comment by patricklodder** [comment]
Thank you! 

I've added `help wanted` because this needs native speaker review.

**Comment by patricklodder** [comment]
Tagging recent French-speaking contributors: @thisiskeanyvy @martyornot @jadehamel @daank-c

Would you have a moment to glance over these newly translated sentences? Would be much appreciated and wow.

**Comment by daank-c** [comment]
@patricklodder, do the string translations in `bitcoin_fr.ts` become the fallback values if the `bitcoin_fr_CA.ts` translations aren't up-to-date?

**Comment by patricklodder** [comment]
> do the string translations in bitcoin_fr.ts become the fallback values if the bitcoin_fr_CA.ts translations aren't up-to-date?

No. `bitcoin_fr_CA.ts` has been unmaintained since day 1 and when we discussed this here on the repo at the end of the last release cycle, the general idea was that we do not update it because it's a lot of double work and use `bitcoin_fr.ts` instead to cover both.

I was planning to propose to delete it for this release - together with every other unmaintained variant - because for `fr` and `fr_CA`, the written language is (as I was told) very much alike - so we can make one really good translation that (presumably) suites all. If that is not the case and there are significant differences, then we can keep it of course, but we should realize that it's double the maintenance work.

**Comment by daank-c** [comment]
> I was planning to propose to delete it for this release - together with every other unmaintained variant - because for `fr` and `fr_CA`, the written language is (as I was told) very much alike

Oh right, thanks for the reminder. Derp

Ok, so these translations were done with a free online translator and make very little sense. I can write the proper translations as a review, or just do it in a different pull-request, up to you.

File: src/qt/locale/bitcoin_fr.ts [hunk 1]
```diff
@@ -1495,15 +1495,15 @@
     </message>
     <message>
         <source>Never share your private key with anyone</source>
-        <translation type="unfinished"></translation>
+        <translation>Ne partagez jamais votre clé privée avec quiconque</translation>
     </message>
     <message>
         <source>Who owns the private keys, owns the coins</source>
         <translation type="unfinished"></translation>
```
Comment (thisiskeanyvy) [inline src/qt/locale/bitcoin_fr.ts:1502]: Celui qui possède les clés privées, possède les pièces

Comment (thisiskeanyvy) [inline src/qt/locale/bitcoin_fr.ts:1502]: Compléter par "Celui qui possède les clés privées, possède les pièces"

File: src/qt/locale/bitcoin_fr.ts [hunk 2]
```diff
@@ -1495,15 +1495,15 @@
...
     </message>
     <message>
         <source>To see ongoing development and contribute, check out the Dogecoin Core repository on GitHub</source>
-        <translation type="unfinished"></translation>
+        <translation>Pour suivre le développement en cours et contribuer, consultez le repo Dogecoin Core sur GitHub.</translation>
```
Comment (thisiskeanyvy) [inline src/qt/locale/bitcoin_fr.ts:1506]: Corriger "repo" du mot anglais repository par "dépôt" qui est plus correct en français, "Pour suivre le développement en cours et contribuer, consultez le dépôt Dogecoin Core sur GitHub."

**Review by thisiskeanyvy — CHANGES_REQUESTED** [review]
One translation to be added and another to be modified for a more correct French otherwise everything is ok for me

File: src/qt/locale/bitcoin_fr.ts (hunk 2 above)
Comment (thisiskeanyvy) [reply src/qt/locale/bitcoin_fr.ts:1506]: Line 1502 can also be completed with "Celui qui possède les clés privées, possède les pièces"

**Comment by omahs** [comment]
@thisiskeanyvy just updated according to your comments

**Comment by patricklodder** [comment]
First off: thanks everyone - you're awesome.

> I can write the proper translations as a review, or just do it in a different pull-request, up to you.

It's really up to you. I would say that if you have additional comments on top of Keany's review, let's just have them here first. If you don't like the result after that, you can always open your own pull request - improvements are always welcome.

**Comment by thisiskeanyvy** [comment]
Would you mind squashing this into a single commit? Then I can approve it, all good to me otherwise.

Feel free to reach out if you'd like any assistance with the squashing. @omahs

**Comment by thisiskeanyvy** [comment]
Use this command to group your commits into one. "git rebase -i HEAD~5" @omahs

**Comment by patricklodder** [comment]
Note: this is now moved to #3148

**Pull request closed without merging**